eq-author-api/db/index.js

```diff
@@ -1,4 +1,45 @@
-var environment = process.env.NODE_ENV || "development";
-var config = require("../config/knexfile.js")[environment];
+const config = require("../config/knexfile.js");
 
-module.exports = require("knex")(config);
\ No newline at end of file
+const getConfig = async function(secretId) {
+  if (secretId && secretId !== "") {
+    const AWS = require("aws-sdk");
+    const client = new AWS.SecretsManager({ region: "eu-west-1" });
+
+    let secretPromise = client.getSecretValue({ SecretId: secretId }).promise();
```

Comment:
This can be `const`

Reply:
This could also be pushed down to the secret definition. So that becomes `const secret = client.getSecretValue...`